homeassistant/components/comfoconnect/sensor.py

```diff
@@ -72,264 +76,266 @@
-        ATTR_LABEL: "Exhaust Humidity",
-        ATTR_UNIT: PERCENTAGE,
-        ATTR_ICON: None,
-        ATTR_ID: SENSOR_HUMIDITY_EXHAUST,
-    },
-    ATTR_AIR_FLOW_SUPPLY: {
-        ATTR_DEVICE_CLASS: None,
-        ATTR_LABEL: "Supply airflow",
-        ATTR_UNIT: VOLUME_FLOW_RATE_CUBIC_METERS_PER_HOUR,
-        ATTR_ICON: "mdi:fan",
-        ATTR_ID: SENSOR_FAN_SUPPLY_FLOW,
-    },
-    ATTR_AIR_FLOW_EXHAUST: {
-        ATTR_DEVICE_CLASS: None,
-        ATTR_LABEL: "Exhaust airflow",
-        ATTR_UNIT: VOLUME_FLOW_RATE_CUBIC_METERS_PER_HOUR,
-        ATTR_ICON: "mdi:fan",
-        ATTR_ID: SENSOR_FAN_EXHAUST_FLOW,
-    },
-    ATTR_BYPASS_STATE: {
-        ATTR_DEVICE_CLASS: None,
-        ATTR_LABEL: "Bypass State",
-        ATTR_UNIT: PERCENTAGE,
-        ATTR_ICON: "mdi:camera-iris",
-        ATTR_ID: SENSOR_BYPASS_STATE,
-    },
-    ATTR_DAYS_TO_REPLACE_FILTER: {
-        ATTR_DEVICE_CLASS: None,
-        ATTR_LABEL: "Days to replace filter",
-        ATTR_UNIT: TIME_DAYS,
-        ATTR_ICON: "mdi:calendar",
-        ATTR_ID: SENSOR_DAYS_TO_REPLACE_FILTER,
-    },
-    ATTR_POWER_CURRENT: {
-        ATTR_DEVICE_CLASS: DEVICE_CLASS_POWER,
-        ATTR_LABEL: "Power usage",
-        ATTR_UNIT: POWER_WATT,
-        ATTR_ICON: None,
-        ATTR_ID: SENSOR_POWER_CURRENT,
-    },
-    ATTR_POWER_TOTAL: {
-        ATTR_DEVICE_CLASS: DEVICE_CLASS_ENERGY,
-        ATTR_LABEL: "Power total",
-        ATTR_UNIT: ENERGY_KILO_WATT_HOUR,
-        ATTR_ICON: None,
-        ATTR_ID: SENSOR_POWER_TOTAL,
-    },
-    ATTR_PREHEATER_POWER_CURRENT: {
-        ATTR_DEVICE_CLASS: DEVICE_CLASS_POWER,
-        ATTR_LABEL: "Preheater power usage",
-        ATTR_UNIT: POWER_WATT,
-        ATTR_ICON: None,
-        ATTR_ID: SENSOR_PREHEATER_POWER_CURRENT,
-    },
-    ATTR_PREHEATER_POWER_TOTAL: {
-        ATTR_DEVICE_CLASS: DEVICE_CLASS_ENERGY,
-        ATTR_LABEL: "Preheater power total",
-        ATTR_UNIT: ENERGY_KILO_WATT_HOUR,
-        ATTR_ICON: None,
-        ATTR_ID: SENSOR_PREHEATER_POWER_TOTAL,
-    },
-}
+@dataclass
+class ComfoconnectRequiredKeysMixin:
+    """Mixin for required keys."""
+
+    sensor_id: int
+
+
+@dataclass
+class ComfoconnectSensorEntityDescription(
+    SensorEntityDescription, ComfoconnectRequiredKeysMixin
+):
+    """Describes Comfoconnect sensor entity."""
+
+    multiplier: float = 1
+
+
+SENSOR_TYPES = (
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_CURRENT_TEMPERATURE,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Inside temperature",
+        unit_of_measurement=TEMP_CELSIUS,
+        sensor_id=SENSOR_TEMPERATURE_EXTRACT,
+        multiplier=0.1,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_CURRENT_HUMIDITY,
+        device_class=DEVICE_CLASS_HUMIDITY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Inside humidity",
+        unit_of_measurement=PERCENTAGE,
```

Comment:
Use `native_unit_of_measurement` instead of `unit_of_measurement` in `ComfoconnectSensorEntityDescription`